scripts/signature.py

```diff
@@ -93,7 +97,50 @@ def diff(args):
         """
         )
 
-    return exit_status
+    return delta, exit_status
+
+
+def diff(args):
+    """Diffs Velox function signatures."""
+    return diff_signatures(args)[1]
+
+
+def bias(args):
+    """Biases a provided fuzzer with newly added functions."""
+
+    delta, status = diff_signatures(args)
+
+    # Return if the signature check call flags incompatible changes.
+    if status:
+        return status
+
+    if not len(delta):
+        print(f"{bcolors.BOLD} No changes detected: Nothing to do!")
+        return 0
+
+    function_set = set()
+    for items in delta.values():
+        for item in items:
+            function_set.add(item.get_root_key())
+
+    # Some functions are prefixed with presto_ , spark_
```

Comment:
Shouldn't we assume here that we have the "raw" function name (without any prefixes)? Relying on the "_" convention could be fragile.

Reply:
No actually since all the functions , whether spark or presto are in one function registry - we add a 'presto_' , or 'spark_' prefix so that we dont clobber functions with similar names. In practice only presto or spark functions are loaded, but in this case we load both so that we know which signatures have been modified. 

However there are some cases where we havent added 'prefix' (or maybe its another issue) when registering the function  and thus these cases the functions come without the 'presto_', 'spark_'. I raised an issue about this : https://github.com/facebookincubator/velox/issues/5670 and will work on it after this diff.

Reply:
Maybe we could export and compare these two sets individually?

```
export_signatures.py --presto > /tmp/presto/contender.txt
export_signatures.py --spark > /tmp/spark/contender.txt

signatures.py --base /tmp/presto/base.txt --contender /tmp/presto/contender.txt > /tmp/presto/bias_string
signatures.py --base /tmp/spark/base.txt --contender /tmp/spark/contender.txt > /tmp/spark/bias_string
```

or something like this.

Reply:
Sg, relatively minor change - Let me make it.